docs/packaging/procedures/release-processes.md

```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
 
 Follow these steps to submit a package for deprecation:
 
-1. Fork and clone the `solus-sc` repo [on GitHub](https://github.com/getsolus/solus-sc), and/or create a new branch
+1. Create a new branch on the monorepo [on GitHub](https://github.com/getsolus/packages) with a suitable name e.g. deprecate-foobar
```

Comment:
```suggestion
1. Create a new branch in the monorepo [on GitHub](https://github.com/getsolus/packages) with a suitable name, for example deprecate-foobar
```
I think the guideline is to use no Latin abbreviations like "e.g." and "i.e."

Should there be a note to fork it, for external contributors?